packages/amplication-git-utils/src/types.ts

```diff
@@ -111,8 +111,8 @@ export interface RemoteGitRepository {
 export interface RemoteGitRepos {
   repos: RemoteGitRepository[];
   total: number | null;
-  currentPage: number | null;
-  pageSize: number | null;
+  page: number | null;
+  perPage: number | null;
 }
```

Comment:
How about creating an interface for pagination? 
So we can extend it across different interfaces yet maintaining the common pattern